src/components/header/template.njk

```diff
@@ -56,38 +55,29 @@
         {% endif %}
       </a>
     </div>
-
-    {% if params.serviceName or params.navigation or params.signOutHref or params.languageToggle %}
+    {% if params.serviceName or params.navigation | length %}
```

Comment:
Do we still need the `params.signOutHref` here?

Reply:
Yes we do, I'll add that back in. Sorry.